internal/graph/check.go

```diff
@@ -704,7 +704,7 @@ func (c *LocalChecker) checkDirect(parentctx context.Context, req *ResolveCheckR
 
 				// if the user value is a typed wildcard and the type of the wildcard
 				// matches the target user objectType, then we're done searching
-				if tuple.IsTypedWildcard(usersetObject) && typesys.GetSchemaVersion() == typesystem.SchemaVersion1_1 {
+				if tuple.IsTypedWildcard(usersetObject) && (typesys.GetSchemaVersion() == typesystem.SchemaVersion1_1 || typesys.GetSchemaVersion() == typesystem.SchemaVersion1_2) {
```

Comment:
This block of code is residual and dead code given that we don't support 1.0 models anymore.

I recommend changing this boolean eval to simply:
```go
if typesystem.IsSchemaVersionSupported(typesys.GetSchemaVersion()) {
  ...
}
```

Reply:
I think having that check remain and do: `t.GetSchemaVersion() == SchemaVersion1_1 || t.GetSchemaVersion() == SchemaVersion1_2)` as @poovamraj suggested above might be worth it.

In this case it would return true for 1.2 only if experimental flag is set